apps/condo/domains/contact/components/CreateContactForm.tsx

```diff
@@ -104,19 +112,24 @@ export const CreateContactForm: React.FC = () => {
         ],
     }
 
-    const { loading, obj: property } = Property.useObject({ where:{ id: selectedPropertyId ? selectedPropertyId : null } })
+    const { loading, data: propertyData } = useGetPropertyWithMapByIdQuery({
+        variables: {
+            id: selectedPropertyId,
+        },
+        skip: !selectedPropertyId,
+        fetchPolicy: 'network-only',
+    })
```

Comment:
Because property after create/update/map_update does not present in property select (or unit in units select), we need to change this behaviour after update client utils on property pages. Will add todo here.